front/src/applications/stdcmV2/components/StdcmOperationalPoint.tsx

```diff
@@ -30,13 +30,13 @@ const StdcmOperationalPoint = ({
   disabled,
 }: StdcmOperationalPointProps) => {
   const { t } = useTranslation('stdcm');
+  const [suggestionsVisible, setSuggestionsVisible] = useState(false);
 
   const { searchTerm, chCodeFilter, sortedSearchResults, setSearchTerm, setChCodeFilter } =
     useSearchOperationalPoint({ initialSearchTerm: point?.name, initialChCodeFilter: point?.ch });
 
   const operationalPointsSuggestions = useMemo(
```

Comment:
I think this `useMemo` is missing a dependency on `searchTerm`?

(With that fixed, maybe we don't need `suggestionsVisible` at all?)

Reply:
Tried adding the missing dependencies on operationalPointsSuggestions and sortedChOptions, the behavior of the input remaining in memory is still here.